src/main/java/roomescape/handler/GlobalExceptionHandler.java

```diff
@@ -39,4 +40,9 @@ public ResponseEntity<List<ExceptionDto>> handleMethodArgumentNotValidException(
                 .toList();
         return ResponseEntity.badRequest().body(exceptionDtos);
     }
+
+    @ExceptionHandler(Exception.class)
+    public ResponseEntity<Void> handleException() {
+        return ResponseEntity.internalServerError().build();
+    }
```

Comment:
공통된 예외 포맷이라면 여기서 메시지를 던져주어도 되겠네요!
```suggestion
    @ExceptionHandler(Exception.class)
    public ResponseEntity<Void> handleException() {
        return ResponseEntity.internalServerError()
            .body(new ExceptionDto("서버에 ~~."));
    }
```